client/my-sites/hosting/staging-site-card/staging-site-card.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import { Card, Gridicon } from '@automattic/components';
+import { action } from '@storybook/addon-actions';
+import { Story, Meta } from '@storybook/react';
+import { useTranslate } from 'i18n-calypso';
+import { ComponentProps } from 'react';
+import { documentHeadStoreMock, ReduxDecorator } from 'calypso/__mocks__/storybook/redux';
+import CardHeading from 'calypso/components/card-heading';
+import { NewStagingSiteCardContent } from './card-content/new-staging-site-card-content';
+
+/**
+ * Ideally, this component should depend only on local `./style.scss`. However, currently, some card styles are defined
+ * in hosting page CSS file, so we need to include it here.
+ */
+import '../style.scss';
+
+export default {
+	title: 'client/components/StagingSite',
+	component: NewStagingSiteCardContent,
+	decorators: [
+		( Story ) => {
+			return (
+				<ReduxDecorator store={ { ...documentHeadStoreMock } }>
+					<Story></Story>
+				</ReduxDecorator>
+			);
+		},
+		( Story ) => {
+			const translate = useTranslate();
+			return (
+				<Card className="staging-site-card">
+					{
+						// eslint-disable-next-line wpcalypso/jsx-gridicon-size
+						<Gridicon icon="science" size={ 32 } />
+					}
+					<CardHeading id="staging-site">{ translate( 'Staging site' ) }</CardHeading>
+					<Story></Story>
+				</Card>
+			);
+		},
+		( Story ) => {
+			return (
+				<div className="hosting" style={ { padding: '20px', backgroundColor: '#f6f7f7' } }>
+					<Story></Story>
+				</div>
+			);
+		},
```

Comment:
Why using three decorators instead of just one?
```tsx
				<div className="hosting" style={ { padding: '20px', backgroundColor: '#f6f7f7' } }>
					<Card className="staging-site-card">
						{
							// eslint-disable-next-line wpcalypso/jsx-gridicon-size
							<Gridicon icon="science" size={ 32 } />
						}
						<CardHeading id="staging-site">{ translate( 'Staging site' ) }</CardHeading>
						<ReduxDecorator store={ { ...documentHeadStoreMock } }>
							<Story></Story>
						</ReduxDecorator>
					</Card>
				</div>
```

Reply:
I wanted to split those three responsibilities:
- first one wraps the story with redux mock, so it's a change allowing use to use component with mocked store
- the second one is a wrapper copied from the main staging site card component, and I wanted to keep it separate to make any further synchronization easier. However, I don't like that much, I would prefer to follow DRY principle here somehow. I opened https://github.com/Automattic/wp-calypso/pull/77153 to extract this wrapper too.
- the third adds styling that is storybook-specific